acms-react-app/.env.development

```diff
@@ -0,0 +1 @@
+REACT_APP_BACKEND_URL=http://localhost:8080
```

Comment:
Good Job adding the URL as an environment variable and separating development from production variables as well